tvl_backends/tvl-backends-fffr/ext/bindings/TvFFFrameReader.cpp

```diff
@@ -175,39 +177,36 @@ uint8_t* TvFFFrameReader::read_frame()
 
 int64_t TvFFFrameReader::read_frames_by_index(int64_t* indices, const int n_frames, uint8_t** frames)
 {
-    int32_t block_size = 8;
-    for (int32_t pos = 0; pos < n_frames;) {
+    int32_t pos = 0;
+    for (; pos < n_frames;) {
         const auto start = &indices[pos];
-        const auto last = pos + block_size;
+        const auto last = pos + static_cast<int32_t>(bufferLength);
```

Comment:
It would be better to get the buffer length from the decoder options held by the Stream instance. Maybe something like `_stream->getBufferLength()`?

Reply:
Done